sklearn/decomposition/tests/test_dict_learning.py

```diff
@@ -789,15 +709,6 @@ def test_update_dict():
     assert_allclose(newd_batch, newd_online)
 
 
-# default value of batch_size changed. FIXME: remove in 1.3
-@pytest.mark.filterwarnings("ignore:The default value of batch_size will change")
-@pytest.mark.parametrize("Estimator", [DictionaryLearning, MiniBatchDictionaryLearning])
-def test_warning_default_transform_alpha(Estimator):
-    dl = Estimator(alpha=0.1, max_iter=5)
-    with pytest.warns(FutureWarning, match="default transform_alpha"):
-        dl.fit_transform(X)
-
-
 # FIXME: remove in 1.3
```

Comment:
Why is this test removed? May you revert the removal?

Reply:
@Micky774 Please have a look

Reply:
Can you instead replace the test in the same position it was originally? This minimizes the git diff, and makes tracking meaningful content changes via `git blame` significantly easier. We favor avoiding unnecessary additions/deletions, e.g. from moving code.

Reply:
@Micky774 How is it now?

Reply:
Test reversion looks good, thanks!

Reply:
@Micky774 There is one thing bugging me. I made most of my changes using min yet it somehow resulted in conflicts. Also, most of conflicts aroused after I use `pre-commit install` and `pre-commit run --all-files`. I know my git skills are weak right now, but still it is a bit strange. 

Also, is this pull request good to go?
Thanks for your help. I know I bothered you a lot